pages/client-libraries/c-sharp.mdx

```diff
@@ -4,19 +4,49 @@ description: Learn to work with Memgraph using C#. Our documentation takes you t
 ---
 
 import { Callout } from "nextra/components";
+import CodeSnippet from '/components/code-snippet/CodeSnippet'
 
 # C# quick start
 
 At the end of this guide, you will have created a simple . NET console **`Hello,
 World!`** program that connects to the Memgraph database and executes simple
 queries.
 
-## Prerequisites
+## Run Memgraph 
```

Comment:
Leave Prerequisites as title, since this client library doesn't have the same structure as others (TODO)